compiler/rustc_target/src/target_features.rs

```diff
@@ -598,7 +598,12 @@ const S390X_FEATURES_FOR_CORRECT_VECTOR_ABI: &'static [(u64, &'static str)] = &[
 const RISCV_FEATURES_FOR_CORRECT_VECTOR_ABI: &'static [(u64, &'static str)] =
     &[/*(64, "zvl64b"), */ (128, "v")];
 // Always warn on SPARC, as the necessary target features cannot be enabled in Rust at the moment.
-const SPARC_FEATURES_FOR_CORRECT_VECTOR_ABI: &'static [(u64, &'static str)] = &[/*(128, "vis")*/];
+const SPARC_FEATURES_FOR_CORRECT_VECTOR_ABI: &'static [(u64, &'static str)] = &[/*(64, "vis")*/];
```

Comment:
Based on https://github.com/rust-lang/rust/issues/131800#issuecomment-2473871620 the situation is actually even weirder than what is described here, but it seems like we can cross that bridge when we come to it, still. The main purpose here of the comment was always to leave something to hit during grepping the codebase.